internal/client/transaction.go

```diff
@@ -0,0 +1,156 @@
+package client
+
+import (
+	"net"
+	"sync"
+	"time"
+
+	"github.com/pion/stun"
+)
+
+const (
+	maxRtxInterval time.Duration = 1600 * time.Millisecond
+)
+
+// TrResult is a bag of result values of a transaction
+type TrResult struct {
```

Comment:
Maybe `TransactionResult` is better, just to remove any ambiguity.

Reply:
True. I will use TransactionResult.

Reply:
Done